docs/DEVELOPER_GUIDE.md

```diff
@@ -321,7 +321,7 @@ Please make sure not to deploy the inferenceservice in the `kfserving-system` or
 
 This often happens when KNative fails to probe the Istio ingress gateway to your inference service and you may find the HTTP error code in KNative `network-istio` pod logs.
 
-If you are seeing HTTP 401 or 302, then you may have Auth turned on for `Istio Ingress Gateway` which blocks the Knative probes to the your service.
+If you are seeing HTTP 401 or 302, then you may have Auth turned on for `Istio Ingress Gateway` which blocks the Knative probes to your service.
```

Comment:
Would it make sense to compile these also in debugging/troubleshooting guide - in one README?

Reply:
I will leave here only for installation trouble shooting, inference service debugging is in the new readme and I will move relevant items there.